worldedit-core/src/main/java/com/sk89q/worldedit/function/entity/ExtentEntityCopy.java

```diff
@@ -142,14 +155,15 @@ private BaseEntity transformNbtData(BaseEntity state) {
                         .putInt("TileZ", newTilePosition.getBlockZ());
 
                 if (hasFacing) {
-                    Direction direction = MCDirections.fromHanging(tag.asInt("Facing"));
+                    boolean isPainting = state.getType() == EntityTypes.PAINTING; // Paintings are very special
+                    Direction direction = isPainting ? MCDirections.fromPre13Hanging(tag.asInt("Facing")) : MCDirections.fromHanging(tag.asInt("Facing"));
```

Comment:
We need to fix this for item frames as well. They use the same horizontal-hanging value.

Reply:
Also, it would probably be a good idea to rename it from `Pre13Hanging` to `HorizontalHanging`, since that's what it actually is.

Reply:
No item frames do not use those values. Just paintings still use the old values,

Reply:
Hmm, it looks like item frames override it. But Leash Knots do not, don't know if we want to handle those @me4502 @wizjany

Reply:
There's a chance mods add entities which extend these (paintings/item frames) and have the same caveats. Unfortunately beyond having some entity property that tells us whether a Hanging entity uses a 4- or 6- value Facing, we probably can't handle all those cases.
That said, we *do* have isPainting and isItemFrame, so perhaps we should prefer those over `== EntityTypes.XYZ` in case mods do extend. (If they change behavior, we will be wrong either way - I think checking extends is "safer").

Reply:
There is no way instead of checking the type, because ClipboardEntities are not faceted.

For leash knots I don't know - currently they cannot be correctly pasted at all, so I cannot test that. And I do not think it is in the scope of this PR to fix that.